package.json

```diff
@@ -17,13 +17,13 @@
     "winston": "2.2.0",
     "express": "4.16.0",
     "body-parser": "1.18.2",
-    "express-winston": "1.3.0",
+    "express-winston": "2.5.1",
     "integrator-extension": "0.3.10"
```

Comment:
Shivam, Please update the integrator-extension version to "0.3.11" and then commit.